suite-native/app/e2e/pageObjects/settingsActions.ts

```diff
@@ -5,15 +5,14 @@ import { TREZOR_E2E_DEVICE_LABEL, scrollUntilVisible } from '../utils';
 import { onAlertSheet } from './alertSheetActions';
 
 class SettingsActions {
-    async tapLocalization() {
-        const localizationElement = element(by.id('@settings/localization'));
-
-        await waitFor(localizationElement).toBeVisible().withTimeout(10000);
-        await localizationElement.tap();
+    async tapPreferences() {
+        const preferencesSettingsElement = element(by.id('@settings/preferences'));
+        await waitFor(preferencesSettingsElement).toBeVisible().withTimeout(10000);
```

Comment:
nit: to align it with other names of this file
```suggestion
        await waitFor(preferencesElement).toBeVisible().withTimeout(10000);
```

Reply:
https://github.com/trezor/trezor-suite/pull/19780/commits/94ca5bfba72ef1c548c06bfa1a60f3c8a3ddf6c0